recipe/meta.yaml

```
@@ -1,18 +1,18 @@
 {% set name = "safetensors" %}
-{% set version = "0.3.2" %}
+{% set version = "0.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 2dbd34554ed3b99435a0e84df077108f5334c8336b5ed9cb8b6b98f7b10da2f6
+  sha256: b985953c3cf11e942eac4317ef3db3da713e274109cf7cfb6076d877054f013e
 
 build:
   number: 0
   # 2023/8/23: On win64 `rust_compiler_rust-gnu` tries to create two artifacts with different hashes in the artifact's filename, and testing one of them fails but another succeeded.
-  skip: True  # [py<37 or (win and (rust_compiler == 'rust-gnu'))]
+  skip: True  # [py<37 or s390x or (win and (rust_compiler == 'rust-gnu'))]
```

```suggestion
  skip: True  # [py<37 or s390x)]
```
rust_gnu on win is only needed if we explicitly add it to the build section, see https://github.com/AnacondaRecipes/aggregate/commit/b2df3cfe1fcb1cb892b893a13bdcbe252182e3c1